src/util/payee-name.js

```diff
@@ -5,12 +5,13 @@ function formatPayeeIban(iban) {
 }
 
 export const formatPayeeName = (trans) => {
+  const amount = trans.amount || trans.transactionAmount.amount;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏do we expect `trans.amount` to be `undefined`? If yes - then using `??` would be more appropriate.

```suggestion
  const amount = trans.amount ?? trans.transactionAmount.amount;
```

Reply:
```javascript
console.log(0 ?? 12); // 0
console.log(0 || 12); // 12

console.log(undefined ?? 12); // 12
console.log(undefined || 12); // 12
```

Reply:
Fixed, turns out `trans.amount` is never set - I missed it when I ported the util over from actual.